src/ClassAliasLoader.php

```diff
@@ -172,9 +172,11 @@ protected function getOriginalClassName($aliasOrClassName)
     protected function loadOriginalClassAndSetAliases($originalClassName)
     {
         if ($this->classOrInterfaceExists($originalClassName) || $this->loadClass($originalClassName)) {
-            foreach ($this->aliasMap['classNameToAliasMapping'][$originalClassName] as $aliasClassName) {
-                if (!$this->classOrInterfaceExists($aliasClassName)) {
-                    class_alias($originalClassName, $aliasClassName);
+            if(is_array($this->aliasMap['classNameToAliasMapping'][$originalClassName])) {
```

Comment:
one minor remark: Can you add a space between the "if" and the "(" so it is consistent? We badly need some CI checks in this package ;)

Reply:
Done.